### DISCUSSION TITLE
rendering Vue/JSX files and serve them with Nitro

### DISCUSSION THREAD

**Discussion by kentntwari** [nitrojs/nitro#2665] [General]
Hi there,

I would like to know if there is a way to render .vue files or jsx from the server, kind of like what Hono does with JSX. I know it would be easier to just use Nuxt but I'm trying to experiment using just Nitro/h3 to do just this.

**Comment by sandros94** [comment]
This becomes particularly useful for sending emails from Nitro side while using Vue components

**Reply by pi0** [reply]
we might support a small util for this soon alongside v3 :)
